docs/spark/tutorials/get-started-macos-instructions.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Get started with .NET for Apache Spark on macOS
+description: Discover how to run a .NET for Apache Spark app using .NET Core on MacOSX.
+ms.date: 01/22/2019
+ms.topic: tutorial
+ms.custom: mvc
+# Customer intent: As a developer, I want to write a simple custom application using .NET for Apache Spark on MacOSX.
```

Comment:
```suggestion
# Customer intent: As a developer, I want to write a simple custom application using .NET for Apache Spark on MacOS.
```

Reply:
I have changed this to `macOS` to be consistent. Thanks!